source/index.html

```diff
@@ -43,6 +43,12 @@
         conjunction with a FIPS capable version of OpenSSL (1.0.2 series). A
         new FIPS module is currently in development.</p>
 
+        <p>OpenSSL 3.0 is the next major version of OpenSSL that is currently
+        in development and includes the new FIPS Object Module. A pre-release
+        version of this is available below. This is for testing only. It should
+        not be used for security criticial purposes. Information and notes
```

Comment:
There is a typo in critical

Reply:
I would even suggest `s/for security criticial/in production/`.

Reply:
Fixed as per @t8m's suggestion